pulp.spec

```diff
@@ -409,6 +409,7 @@ Pulp provides replication, access, and accounting for software repositories.
 %{_bindir}/pulp-manage-db
 %{_bindir}/pulp-qpid-ssl-cfg
 %{_bindir}/pulp-gen-ca-certificate
+%{_bindir}/pulp_streamer
```

Comment:
Oh right the file needs to be declared that the package owns it. Thanks for the answer!